Modules/_csv.c

```diff
@@ -457,7 +465,7 @@ dialect_new(PyTypeObject *type, PyObject *args, PyObject *kwargs)
         goto err;
     if (self->delimiter == 0) {
         PyErr_SetString(PyExc_TypeError,
-                        "\"delimiter\" must be a 1-character string");
+                        "delimiter must be set");
```

Comment:
I think "set" might confuse naive readers. What was wrong with the previous spelling? Also, note that "delimiter" was quoted because it was explicitly referring to the "dellimiter" attribute.

Reply:
Okay, I agree that the message doesn't need to be changed.

Reply:
It was made consistent with error messages for other arguments.